database/migrations/20181126185213_users.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('users', users => {
+    users.increments();
+
+    users
+      .string('username', 128)
+      .notNullable()
+      .unique();
+    users.string('password', 128).notNullable();
```

Comment:
I recommend keeping the syntax uniform throughout the file.  For example, string and notNullable should be dropped down to lines 11 and 12 as it is above.